lib/zendesk_apps_tools/command.rb

```diff
@@ -173,6 +174,7 @@ def server
         server.set :public_folder, File.join(options[:path], 'assets')
         server.set :parameters, settings
         server.set :app_id, options[:app_id]
+        server.set :plan, [options[:plan], cache.fetch('plan')].reject(&:nil?).first
```

Comment:
Hello Fabio, this line in interesting for me. I'll ask you about this in 🍗 when you are free.